### PR TITLE
Move checkboxes and text fields down to submit button

### LINKED ISSUE #938: Move "minor edit" and "edit comments" close to the Save button
The editing field is to huge now, and if you're doing a lot of edits, it becomes incredibly annoying to jump back and forth with the mouse, hitting things on the opposite sides of the screen, and in my case I even have to scroll up and down (900p monitor).

### PR DESCRIPTION
**Pull request by Masterjun3**
Resolves #938.

Changes include minor edits box, watch topic box, and revision message field.

(before | after)
image

### CODE REVIEW

File: TASVideos/Pages/Forum/Posts/Create.cshtml [hunk 1]
```diff
@@ -20,17 +20,17 @@
 			<span asp-validation-for="Post.Mood" class="text-danger"></span>
 		</column>
 	</row>
-	<fullrow class="mt-3">
```
Comment (adelikat) [inline TASVideos/Pages/Forum/Posts/Create.cshtml:23]: why did this need to change?

**Comment by TiKevin83** [comment]
I like the use of the form grouping, it looks cleaner to me that way though in that screenshot something is off with the vertical centering of the checkbox

**Comment by Masterjun3** [comment]
Yeah apparently [bootstrap itself](https://getbootstrap.com/docs/5.1/forms/input-group/#checkboxes-and-radios) suggests adding  `mt-0`.

**Comment by vadosnaprimer** [comment]
Sometimes when making [important changes](https://tasvideos.org/Wiki/PageHistory?path=MovieRules%2FLegacy) to the wiki, I need to write elaborate edit notes. So having more space in "edit comments" is important. Can the width of that field be made to fill all available width?

**Comment by Masterjun3** [comment]
They go to full width when you make your width smaller. :) :)

**Comment by Masterjun3** [comment]
Real answer, it doesn't look good at full width in large screens.
Ideally we would have automatically resizing text fields, giving you multiple lines when you write a lot. But I think that's another unsolved html problem and javascript might be needed again.

**Comment by vadosnaprimer** [comment]
In terms of taste, it would just be as wide as the main edit field, I don't see that as a problem. I think being functionally helpful is more important here than looking nice and tiny.

**Comment by TiKevin83** [comment]
I think a good compromise would be `col-xl-8 col-xxl-6` instead of `col-lg-6`

**Comment by Masterjun3** [comment]
It's now full width.

**Review by vadosnaprimer — APPROVED** [review]
Thank you!

**Pull request merged**